float64[16] NE_T_EE
float64[16] EE_T_K
---
bool success

